composer.json

```diff
@@ -16,5 +15,11 @@
   "support": {
     "issues": "https://github.com/Automattic/WPCOM-Legacy-Redirector/issues",
     "source": "https://github.com/Automattic/WPCOM-Legacy-Redirector"
+  },
+  "require-dev": {
+    "automattic/vipwpcs": "^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
+    "phpunit/phpunit": "^7"
```

Comment:
Likewise: `^4 || ^5 || ^6 || ^7` gives more flexibility about which version is installed. Until there is something just from PHPUnit 7 which is being used (and can then likely use https://github.com/Yoast/PHPUnit-Polyfills anyway to make the PHPUnit 7 thing available in earlier versions anyway), then keep the constraint as loose as possible (then supports PHP 5.6 as well).